### ISSUE TITLE
Inconsistent references to datadir/datarootdir

### ISSUE THREAD

**Issue by paravoid** [maxmind/geoipupdate-legacy#86]
`bin/Makefile.am` and `man/Makefile.am` refer to `$(datadir)/GeoIP"`, e.g.
```
AM_CPPFLAGS = -I$(top_srcdir)/src -DSYSCONFDIR=\"$(sysconfdir)\" -DDATADIR=\"$(datadir)/GeoIP\" \
                          -std=c99 -Wall -Wextra
```

but `Makefile.am` uses `$(datarootdir)/GeoIP`:
```
$(MKDIR_P) "$(DESTDIR)$(datarootdir)/GeoIP"
```

These two directories may or may not be the same, depending on autoconf defaults and `./configure` arguments. (FWIW, in the Debian package, I've been passing `--datadir=/var/lib` to configure, as I want the files to end up in `/var/lib/GeoIP` rather than somewhere /usr, which is for read-only data)

**Comment by horgh** [comment]
Interesting. It looks like we should use `$(datadir)` in `Makefile.am`.

From reading docs, `$(datadir)` seems more appropriate. (["The directory for installing idiosyncratic read-only architecture-independent data files for this program. This is usually the same place as ‘datarootdir’, but we use the two separate variables so that you can move these program-specific files without altering the location for Info files, man pages, etc."](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html))

If we changed to use `$(datarootdir)` in the other spots, it would mean we'd have to specify `--datarootdir` to override the default database directory, and that would impact stuff like `$(mandir)` as well.

Thank you for reporting this!

**Comment by horgh** [comment]
This should be fixed with #94.

Thanks again!
